buildspec.yml

```
@@ -0,0 +1,17 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - n 18
```

```suggestion
      - n 20
```

And on that note, maybe we should add an `engines` property to the package.json.